server/api.js

```diff
@@ -196,6 +230,42 @@ router.put("/graduates/:id", function (req, res) {
     ],
     (error, result) => {
       if (result) {
+        Connection.query(
```

Comment:
this nested code can/should be enhanced, in order to improve the readability, I understand it works now, and you're in rush, but 2 weeks from now you won't even remember how it works, also this spaghetti code will drive you crazy if you have a bug somewhere in it